website/static/js/pages/prereg-admin-page.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+var $osf = require('js/osfHelpers');
+var ko = require('knockout');
+var $ = require('jquery');
+var Raven = require('raven-js');
+
+var drafts;
+
+ko.bindingHandlers.enterkey = {
+    init: function (element, valueAccessor, allBindings, viewModel) {
+        var callback = valueAccessor();
+        $(element).keypress(function (event) {
+            var keyCode = (event.which ? event.which : event.keyCode);
+            if (keyCode === 13) {
+                callback.call(viewModel);
+                return false;
+            }
+            return true;
+        });
+    }
+};
+
+function adminView(data) {
```

Comment:
Looking at this view model as a whole and thinking about what it needs to accomplish there's a couple big picture comments:
- this VM should fetch and store a list of drafts
- drafts should have their own nested view model for editing draft information
  - you might consider encapsulating the save/update logic in this VM